foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_11_0/HiveUniversalConfigYAMLMigration.java

```diff
@@ -0,0 +1,110 @@
+package com.boozallen.aissemble.upgrade.migration.v1_11_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.util.YamlUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.error.YAMLException;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+
+public class HiveUniversalConfigYAMLMigration extends AbstractAissembleMigration {
+    private static final Logger logger = LoggerFactory.getLogger(HiveUniversalConfigYAMLMigration.class);
+    private static final List<String> propertiesForConfigStore = Arrays.asList("javax.jdo.option.ConnectionUserName");
+    private static final List<String> mysqlAuthForConfigStore = Arrays.asList("username");
+    private static final HashSet<String> linesToRemove = new HashSet<>(Arrays.asList("username: hive", "- name: javax.jdo.option.ConnectionUserName" ,"value: hive", "description: Username to use against metastore database", "# NB: UCS"));
+    private static final String DEFAULT_HIVE_USERNAME_AND_PASSWORD = "hive";
+    @Override
+    protected boolean shouldExecuteOnFile(File file) {
+        if (file.getName().endsWith("values.yaml")) {
```

Comment:
This check is redundant as baton is only going to select `values.yaml` files due to your configuration in the `migrations.json`

Reply:
Make sense. Removed the redundant check!